docs/dev/cloudFoundry/CloudFoundryPlugin_Basics.md

```diff
@@ -0,0 +1,42 @@
+# Cloud Foundry Plugin
+
+## Basics of Cloud Foundry
+Cloud Foundry is a open-source Platform as a Service Model (PaaS) which allows everyone to deploy easily an existing application on an infrastructure e.g. AWS, vSphere or OpenStack.
+There are two types of possible use-cases:
+1. deploy an application on a infrastructure with existing underlying CloudFoundry-Platform --> [CF-Cloud-Provider](https://www.cloudfoundry.org/certified-platforms/)
+2. deploy on an IaaS like AWS, vSphere, or OpenStack. Therefore you have to install CloudFoundry manually.
```

Comment:
I would ignore this use-case for now. 

We want to transform the application onto cloud foundry and not try to deploy something on aws (for example) using CloudFoundry.
If there are no (major) api differences between different platforms the user just supplies the endpoint (when deploying) and the Transformator handles the rest.

Reply:
thanks for your comment

i wanted to give a short overview of what cloud foundry is. It is only background information for our project. But maybe it isn´t necessery.

in this case the transfomator have to deploy with the cloud foundry api or using the CLI commands. I think this is stuff for the special Plugin, isn´t it?